addons/website/static/src/builder/plugins/options/animate_text.scss

```diff
@@ -2,4 +2,11 @@
     min-width: unset;
     width: 300px;
     overflow-y: auto;
+
+    // Apply 10px to the visible sublevel when it's the 2nd child,
+    // or to the 3rd child when the 2nd child is hidden (.d-none).
+    & .hb-row:nth-child(2).hb-row-sublevel-1:not(.d-none) > .hb-row-label::after,
```

Comment:
We already have css for this, why we need to write css explicitly?
<img width="570" height="151" alt="image" src="https://github.com/user-attachments/assets/8018975b-a08f-4c3a-bd1f-5a20bd61558f" />

Reply:
@msh-odoo Yes but that rule is used across builder options and must not be altered. Per the spec, the connector in the animate-text popover requires a different vertical offset only within that popover. To avoid unintended side effects we add a scoped rule under .o_animate_text_popover that sets the ::after top value specifically for the popover, leaving the global builder option styles unchanged.

<img width="1045" height="677" alt="image" src="https://github.com/user-attachments/assets/15ea7ca5-724a-409e-9f17-ab3428c4d402" />